### PR TITLE
[numpy][tvm] rad2deg && deg2rad

### PR DESCRIPTION
**Pull request by tingying2020**
Numpy operator rad2deg and deg2rad implemented with tam.

@haojin2

### CODE REVIEW

File: python/mxnet/ndarray/numpy/_op.py [hunk 1]
```diff
@@ -32,7 +32,8 @@
            'exp', 'expm1', 'arcsin', 'arccos', 'arctan', 'sign', 'log', 'degrees', 'log2', 'log1p',
            'rint', 'radians', 'reciprocal', 'square', 'negative', 'fix', 'ceil', 'floor',
            'trunc', 'logical_not', 'arcsinh', 'arccosh', 'arctanh', 'tensordot',
-           'linspace', 'expand_dims', 'tile', 'arange', 'split', 'concatenate', 'stack']
+           'linspace', 'expand_dims', 'tile', 'arange', 'split', 'concatenate', 'stack', 'deg2rad',
+           'rad2deg']
```
Comment (haojin2) [inline python/mxnet/ndarray/numpy/_op.py]: move 'deg2rad' to after 'radians' and 'rad2deg' to after 'degrees'.

Comment (haojin2) [reply python/mxnet/ndarray/numpy/_op.py]: Same for the func declarations within the file

Comment (haojin2) [reply python/mxnet/ndarray/numpy/_op.py]: In all files.

File: tests/python/unittest/test_numpy_op.py [hunk 2]
```diff
@@ -1080,6 +1080,80 @@ def hybrid_forward(self, F, a, *args):
...
+
+    types = ['float64', 'float32']
+    for hybridize in [True, False]:
+        for shape in [(),
+                      (1,),
+                      (1, 1),
+                      (1, 2, 3),
+                      (1, 0),
+                      (2, 0, 3)
+                      ]:
```
Comment (haojin2) [inline tests/python/unittest/test_numpy_op.py]: simply do
```Python
for shape in [(), (1,), (1, 1), (1, 2, 3), (1, 0), (2, 0, 3)]:
```

File: python/mxnet/ndarray/numpy/_op.py (hunk 1 above)
Comment (tingying2020) [reply python/mxnet/ndarray/numpy/_op.py]: Fixed. Thank you.

File: tests/python/unittest/test_numpy_op.py (hunk 2 above)
Comment (tingying2020) [reply tests/python/unittest/test_numpy_op.py]: Have put the test into test_np_unary_funcs. Thanks.

File: src/operator/contrib/tvmop/ufunc.cc [hunk 3]
```diff
@@ -144,6 +144,118 @@ NNVM_REGISTER_OP(_backward_contrib_tvm_vadd)
...
+inline bool DegandradOpType(const nnvm::NodeAttrs& attrs,
+                            std::vector<int>* in_attrs,
+                            std::vector<int>* out_attrs) {
+  CHECK_EQ(in_attrs->size(), 1U);
+  CHECK_EQ(out_attrs->size(), 1U);
+
+  TYPE_ASSIGN_CHECK(*out_attrs, 0, in_attrs->at(0));
+  TYPE_ASSIGN_CHECK(*in_attrs, 0, out_attrs->at(0));
+  CHECK_LE(in_attrs->at(0), mshadow::kFloat64)
+    << "Only support float32 and float64.";
```
Comment (haojin2) [inline src/operator/contrib/tvmop/ufunc.cc]: You should have a different rule for checking this, the ids of data types could be easily changed and this rule will be broken.

**Comment by haojin2** [comment]
Almost LGTM, @yzhliu for double-check.

File: src/operator/contrib/tvmop/ufunc.cc (hunk 3 above)
Comment (tingying2020) [reply src/operator/contrib/tvmop/ufunc.cc]: Changed to 
"CHECK(in_attrs->at(0) == mshadow::kFloat64 ||
               in_attrs->at(0) == mshadow::kFloat32)
    << "Only support float32 and float64.";"

**Comment by haojin2** [comment]
@yzhliu

**Pull request merged**